core/src/main/java/org/apache/accumulo/core/iterators/IterationInterruptedException.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.accumulo.core.iterators;
 
+/**
+ * Exception thrown if an interrupt flag is detected.
+ */
 public class IterationInterruptedException extends RuntimeException {
```

Comment:
Should this RTE exist in public API, or should it be in iteratorsImpl as well?

Reply:
Good catch. I moved it to iteratorsImpl in 892a867. Thankfully, nothing outside of the tserver code and Rfile use it.